src/Tizen.NUI/src/public/BaseComponents/VideoView.cs

```diff
@@ -447,6 +449,54 @@ public string ResourceUrl
             }
         }
 
+        /// <summary>
+        /// Enables auto rotation of the video based on the orientation of the video contents.
+        /// </summary>
+        /// <note> This feature is supported only when underlay is false. </note>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool IsAutoRotationEnabled
```

Comment:
프로퍼티가 get/set 모두 있는데, IsXXX 이렇게 하기 보다는 Is 없이 그냥 AutoRatationEnabled 라고 하는게 좀 더 명확하지 않을까요?
개인적으로, IsXXX 는 get 에만 해당되는, 즉 현재 상태를 나타내는 듯한 뉘앙스가 있어 보입니다.
@rabbitfor 님, @jaehyun0cho 님,
어떻게 생각하시는지요?

Reply:
How about adding Is or somthing similar for prefix? For example, IsAutoRotationEnabled.
I found many methods in TizenFX are using this name pattern for boolean property and also the MSDN framework design guildlines do encourage usage of Is or Has.
https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members#names-of-properties

DO name Boolean properties with an affirmative phrase (CanSeek instead of CantSeek). Optionally, you can also prefix Boolean properties with "Is", "Can", or "Has", but only where it adds value.

=> 이런 내용이 있었군요. 
확인 감사합니다. 이건 닫겠습니다.